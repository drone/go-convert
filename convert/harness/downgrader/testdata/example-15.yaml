kind: pipeline
spec:
  stages:
    - name: example
      spec:
        platform:
          arch: amd64
          os: linux
        runtime:
          spec: {}
          type: cloud
        steps:
          - spec:
              run: echo "Run my tests"
            type: script
          - name: store_test_results
            spec:
              reports:
                - path:
                    - test-results/*.xml
                  type: junit
              run: echo upload unit test results
            type: script
      type: ci
version: 1