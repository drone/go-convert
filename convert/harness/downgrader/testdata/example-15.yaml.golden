pipeline:
  identifier: default
  name: default
  orgIdentifier: default
  projectIdentifier: default
  properties:
    ci:
      codebase:
        build: <+input>
  stages:
  - stage:
      identifier: example
      name: example
      spec:
        cloneCodebase: true
        execution:
          steps:
          - step:
              identifier: script
              name: script
              spec:
                command: echo "Run my tests"
              timeout: ""
              type: Run
          - step:
              identifier: storetestresults
              name: store_test_results
              spec:
                command: echo upload unit test results
                reports:
                  spec:
                    paths:
                    - test-results/*.xml
                  type: JUnit
              timeout: ""
              type: Run
        platform:
          arch: Amd64
          os: Linux
        runtime:
          spec: {}
          type: Cloud
      type: CI
